server/e2e/api/specs/library.e2e-spec.ts

```diff
@@ -194,18 +194,7 @@ describe(`${LibraryController.name} (e2e)`, () => {
         .set('Authorization', `Bearer ${user1.accessToken}`)
         .send({ type: LibraryType.EXTERNAL });
 
-      expect(status).toBe(201);
-      expect(body).toEqual(
-        expect.objectContaining({
-          ownerId: user1.userId,
-          type: LibraryType.EXTERNAL,
-          name: 'New External Library',
-          refreshedAt: null,
-          assetCount: 0,
-          importPaths: [],
-          exclusionPatterns: [],
-        }),
-      );
+      expect(status).toBe(403);
```

Comment:
Can you assert the body message is correct as well?

Reply:
Done